cosmic-panel-config/src/panel_config.rs

```diff
@@ -144,6 +146,7 @@ impl PanelSize {
                 PanelSize::M => 28,
                 PanelSize::L => 32,
                 PanelSize::XL => 48,
+                PanelSize::Custom(s) => *s / 2,
```

Comment:
I think this is good, but maybe it would be good to round to the nearest multiple of 2 or 4 for this calculation and the others like it?